templates/partials/_navigation.html

```diff
@@ -21,84 +21,42 @@
+          <ul class="p-navigation__dropdown" id="forum-link-menu" aria-hidden="true">
             <li>
-              <a class="p-navigation__dropdown-item" href="/cloud-native-kubernetes-usage-report-2022">Kubernetes &amp; Cloud Native Operations Report</a>
+              <a href="https://discourse.charmhub.io/" class="p-navigation__dropdown-item">Discourse forum</a>
             </li>
             <li>
-              <a class="p-navigation__dropdown-item" href="/operator-day">Operator Day</a>
-            </li>
-            <li>
-              <a class="p-navigation__dropdown-item" href="/blog">Blog</a>
-            </li>
-            <li>
-              <a class="p-navigation__dropdown-item" href="https://charmhub.io/topics">Topics</a>
+              <a href="https://chat.charmhub.io" class="p-navigation__dropdown-item">Mattermost chat</a>
             </li>
           </ul>
         </li>
         <li class="p-navigation__item--dropdown-toggle" id="contribute-link">
-          <a class="p-navigation__link" href="#contribute-link-menu" aria-controls="contribute-link-menu">Contribute</a>
-          <ul class="p-navigation__dropdown" id="contribute-link-menu" aria-hidden="true">
-            <li>
-              <div class="p-navigation__dropdown-item is-title">
-                <div class="p-muted-heading u-no-margin--bottom">
-                  Create a charm
-                </div>
-              </div>
-            </li>
-            <li>
-              <a class="p-navigation__dropdown-item" href="/docs/sdk/set-up-a-charm-project">Build</a>
-            </li>
+          <a class="p-navigation__link" href="#docs-link-menu" aria-controls="docs-link-menu">Docs</a>
+          <ul class="p-navigation__dropdown" id="docs-link-menu" aria-hidden="true">
             <li>
-              <a class="p-navigation__dropdown-item" href="/docs/sdk/publishing">Publish</a>
+              <a href="/docs" class="p-navigation__dropdown-item">Juju</a>
             </li>
             <li>
-              <div class="p-navigation__dropdown-item is-title">
-                <div class="p-muted-heading u-no-margin--bottom">
-                  Join the community
-                </div>
-              </div>
+              <a href="/docs/sdk" class="p-navigation__dropdown-item">Charm SDK</a>
             </li>
             <li>
-              <a class="p-navigation__dropdown-item" href="https://discourse.charmhub.io/">Forum</a>
-            </li>
-            <li>
-              <a class="p-navigation__dropdown-item" href="https://chat.charmhub.io/charmhub/channels/juju">Chat</a>
-            </li>
-            <li>
-              <a class="p-navigation__dropdown-item" href="https://bugs.launchpad.net/juju">Report a bug</a>
-            </li>
-            <li>
-              <a class="p-navigation__dropdown-item" href="/careers">Careers</a>
+              <a href="/docs/dev" class="p-navigation__dropdown-item">Contribute to Juju</a>
             </li>
```

Comment:
```suggestion
            </li>
            <li>
              <a href="/operator-day" class="p-navigation__dropdown-item">Operator Day</a>
            </li>
```
Adding this to link to the operator day page https://juju.is/operator-day